cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/1_create_datasource.spec.js

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+// Get environment variables
+const endPointUrl = Cypress.env('OSD_TEST_DOMAIN_ENDPOINT_URL');
+const invalidEndPointUrl = Cypress.env('OSD_INVALID_ENPOINT_URL');
+const username = Cypress.env('username');
+const password = Cypress.env('password');
+
+if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
+  describe('Create datasources', () => {
+    beforeEach(() => {
+      // Visit OSD create page
+      miscUtils.visitPage(
+        'app/management/opensearch-dashboards/dataSources/create'
+      );
+    });
+
+    it('should successfully load the page', () => {
+      cy.contains(
+        'Create a new data source connection to help you retrieve data from an external OpenSearch compatible source.',
+        { timeout: 60000 }
+      );
+    });
+
+    it('should successfully show the experimental feature callout', () => {
```

Comment:
this will be updated when the feature becomes permanent in the future, right? do you have an issue to track it?

Reply:
Correct, will create an issue to track it.